common/lib/xmodule/xmodule/tests/xml/factories.py

```diff
@@ -134,6 +134,13 @@ def parent(self, _create, extracted, **_):
 
         extracted._xml_node.append(self._xml_node)  # pylint: disable=no-member, protected-access
         extracted.policy.update(self.policy)
+        self._xml_node.attrib.update(  # pylint: disable=no-member, protected-access
+            {
+                key: value
+                for key, value in extracted._xml_node.attrib.iteritems()  # pylint: disable=no-member, protected-access
+                if key in InheritanceMixin.fields.keys()
+            }
+        )
```

Comment:
I believe:
The inheritance of fields works through the runtime.  And since the sequence module tests you are adding to uses a mock runtime, the inheritance isn't working.